src/Widgets/PreferencesDialog.vala

```diff
@@ -19,6 +19,45 @@
 * Authored by: Felipe Escoto <felescoto95@hotmail.com>
 */
 
+/**
+ *  The purpose of this class is to easily create a new Switchbox for the
+ *  PreferencesDialog class.
+ */
+private class ENotes.PreferencesDialogSwitchBox {
+
```

Comment:
For the Commit history, i'd rather it be a different commit :smile: (It's one of the things we do at the rest of the elementary projects. It's better to have many small commits than large commits, specially with PRs)